Calculator.Lonchanick/Calculator.Lonchanick/Calculator.cs

```diff
@@ -0,0 +1,59 @@
+﻿
+namespace Calculator.Lonchanick
```

Comment:
You can use this notation for all your namespaces to save one line of indentation:

![image](https://user-images.githubusercontent.com/108711476/211710112-80ceb71f-3286-4db7-9c65-7db1931c5a1b.png)